modules/nf-core/custom/tx2gene/tests/main.nf.test

```diff
@@ -16,68 +16,63 @@ nextflow_process {
             script "../../../untar/main.nf"
             process {
                 """
-                input[0] = [
-                [ id:'test'], // meta map
-                file(params.test_data['saccharomyces_cerevisiae']['genome']['kallisto_results'], checkIfExists: true)
-                ]
+                input[0] = Channel.of([
+                    [ id:'test'], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/eukaryotes/saccharomyces_cerevisiae/kallisto_results.tar.gz', checkIfExists: true)
+                ])
                 """
             }
         }
     }
 
     test("saccharomyces_cerevisiae - gtf") {
-        
+
         when {
             process {
                 """
-                input[0] = [
-                [ id:'test'], // meta map
-                file(params.test_data['saccharomyces_cerevisiae']['genome']['genome_gfp_gtf'], checkIfExists: true)
-                ]
+                input[0] = Channel.of([
+                    [ id:'test'], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/eukaryotes/saccharomyces_cerevisiae/genome_gfp.gtf', checkIfExists: true)
+                ])
                 input[1] = UNTAR.out.untar.map { meta, dir -> [ meta, dir.listFiles().collect() ] }
                 input[2] = 'kallisto'
-                input[3] = 'gene_id' 
+                input[3] = 'gene_id'
                 input[4] = 'gene_name'
                 """
             }
         }
 
-
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out.tx2gene).match('tx2gene') },
-                { assert snapshot(process.out.tx2gene).match('versions') }
+                { assert snapshot(process.out).match() }
```

Comment:
I separated these tests on purpose, to avoid duplication in the .snap!

Reply:
I feel like the second `process.out.tx2gene` should have been a versions.

That being said, I do agree with the fact the the duplication in the snap is not nice, but as it's generated by nf-test, for me it's more something that needs fixing on the nf-test side and then update, rather than us having to adapt.

Reply:
Ahh yes, now you're right on that one- my bad.

But I do also thing this form is more readable.